config/cache.ts

```diff
@@ -0,0 +1,19 @@
+export const CACHE_DEFAULT_HEADERS =
+  'public, max-age=180, stale-if-error=1200, stale-while-revalidate=60';
+
+export const CACHE_ALLOWED_LIST_FILES_PATHS = [
+  { path: /^\/favicon-?[^@]*.(svg|ico)?$/, headers: CACHE_DEFAULT_HEADERS },
+  { path: /^\/manifest.json?$/, headers: CACHE_DEFAULT_HEADERS },
+];
+
+export const findCacheControlFileHeaders = (
+  url?: string,
+): string | undefined => {
+  const requestPath = url?.split('?').shift();
```

Comment:
you can theoretically have url like `test.com/request?param?=1` (it is invalid but it still can work) so better to parse it with `URL.parse`

Reply:
Also, using `[0]` or `.at(0)` is better both for readability and performance, as it is not altering the source array

Reply:
used @lidofinance/next-cache-files-middleware